src/show.jl

```diff
@@ -70,6 +70,11 @@ function print_header(io, Δt, Δb, ∑t, ∑b, name_length, header, allocations
     if header
         time_alloc_pading = " "^(strwidth(sec_ncalls))
 
+        if length(title) > strwidth(sec_ncalls)
+            title = string(title[1:strwidth(sec_ncalls)-3], "...")
```

Comment:
Not that it matters but creating a sting like that is pretty bad because you create all the `length(str) - 1` strings fully. Using an `IOBuffer` is preferred. Anyway, the strings here are short so it doesn't matter.